chimney/src/main/scala-2/io/scalaland/chimney/internal/compiletime/ChimneyTypesPlatform.scala

```diff
@@ -406,6 +417,17 @@ private[compiletime] trait ChimneyTypesPlatform extends ChimneyTypes { this: Chi
       }
     }
 
+    object PathList extends PathListModule {
+      val Empty: Type[runtime.PathList.Empty] = weakTypeTag[runtime.PathList.Empty]
+      object List extends ListModule {
+        def apply[Head <: runtime.Path: Type, Tail <: runtime.PathList: Type]: Type[runtime.PathList.List[Head, Tail]] =
+          weakTypeTag[runtime.PathList.List[Head, Tail]]
+        def unapply[A](A: Type[A]): Option[(?<[runtime.Path], ?<[runtime.PathList])] =
+          A.asCtor[runtime.PathList.List[?, ?]]
+            .map(A0 => A0.param_<[runtime.Path](0) -> A0.param_<[runtime.PathList](1))
+      }
+    }
```

Comment:
A list of paths would not be needed if instead this would apply a config, as many times as many paths there is in vararg. This would make less code to maintain in DSL and macros.